src/wallet/feebumper.cpp

```diff
@@ -195,7 +195,7 @@ Result CreateTransaction(const CWallet* wallet, const uint256& txid, const CCoin
     // If the output would become dust, discard it (converting the dust to fee)
     poutput->nValue -= nDelta;
     if (poutput->nValue <= GetDustThreshold(*poutput, GetDiscardRate(*wallet, ::feeEstimator))) {
-        LogPrint(BCLog::RPC, "Bumping fee and discarding dust output\n");
+        LogPrint(BCLog::RPC, "[%s] Bumping fee and discarding dust output\n", wallet->GetName());
```

Comment:
I think this should just be changed to a `WalletLogPrintf`. I don't think this needs to be in the RPC category.

Reply:
Fixed